src/server/api/schema.js

```diff
@@ -424,6 +424,7 @@ const rootMutations = {
         const newUserData = {
           first_name: capitalizeWord(userData.firstName),
           last_name: capitalizeWord(userData.lastName),
+          alias: capitalizeWord(userData.alias),
```

Comment:
Since alias can easily override firstName -- should we chomp off whitespace before and after (probably good to do for first/last name, if you want, too)

Reply:
Sorry, I didn't fully understand the suggestion here.

Reply:
I'm worried about the case where someone accidentally adds `__` (spaces) into the optional Alias field.  Since it overrides first_name in places, I think it's important to either remove front and leading spaces (so that 'only spaces' alias turns into an empty string and will then evaluate to false), and/or test that in this place.

Since we'd be stripping whitespace, I was thinking that might be good to do, even for the fields first/last name, as well, but that has less of a fatal result.

Reply:
Got it. Will update the PR, trimming all three fields here.

Reply:
@schuyler1d  Just pushed a new version. Check it out and let me know!